src/tab.rs

```diff
@@ -491,6 +497,17 @@ pub fn scan_path(tab_path: &PathBuf, sizes: IconSizes) -> Vec<Item> {
         (false, true) => Ordering::Greater,
         _ => LANGUAGE_SORTER.compare(&a.display_name, &b.display_name),
     });
+    items.iter_mut().for_each(|mut item| {
+        if item.path_opt().is_some() {
```

Comment:
I would recommend to use if let Some(path) = item.path_opt() here, then use hidden.eq(path) below.